ydb/docs/en/core/best_practices/_includes/secondary_indexes.md

```diff
@@ -82,6 +88,12 @@ FROM series VIEW views_index
 WHERE views = 0;
 ```
 
+{% note info %}
+
+Currently, data deletion is possible using a synchronous secondary index only.
```

Comment:
```suggestion
Currently, deleting data is possible only using a synchronous secondary index. This is because data removal is possible only in [Serializable](../concepts/transactions.md#modes) transactions, but accessing asynchronous indices would break the guarantees of this transaction mode.
```